common/rdm/RDMHelper.cpp

```diff
@@ -994,15 +994,15 @@ string StatusMessageIdToString(uint16_t message_id,
       // Data Value shall be a signed integer." but I'm sure it's what was
       // intended. The same thing is technically true with the slots too.
       str << "Proxy Drop: PID "
-          << ToHex(reinterpret_cast<uint16_t&>(data1)) << " at TN "
+          << strings::ToHex(reinterpret_cast<uint16_t&>(data1)) << " at TN "
```

Comment:
Either way, I tend to switch to using when there are many occurrences in one file. Many is subjective though :)

We don't need ola::strings::ToHex since we're already in the ola namespace.